src/components/SocialListItem.tsx

```diff
@@ -27,32 +25,30 @@ const icons = {
   webpage: FaGlobe,
 }
 
-export interface IProps {
+export type SocialListItemProps = {
```

Comment:
what's the reason to change this from interface to type?

Reply:
Jump start on our discussion earlier about updated our TypeScript standards

[#11421](https://github.com/ethereum/ethereum-org-website/issues/11421) (Canonical repo)
> Thirdly, preference for the component props type should be a type and not an interface (with specific exceptions) as noted in the [React-Typescript cheatsheet](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example#types-or-interfaces). Because this project is not a 3rd-party API with components, prefer type for the added constraint because we are not trying to do something like [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces).

Same reason for renaming